src/middleware/navigationMiddleware.ts

```diff
@@ -4,35 +4,49 @@ import * as url from "node:url";
 import logger from "../lib/Logger";
 import { Navigation } from "types/navigation";
 import { getRemoveMemberCheckDetailsFullUrl } from "../lib/utils/urlUtils";
+import { UserRole, AcspMembership } from "private-api-sdk-node/dist/services/acsp-manage-users/types";
+import { getExtraData } from "../lib/utils/sessionUtils";
 
 export const NAVIGATION: Navigation = {
     [constants.CHECK_MEMBER_DETAILS_FULL_URL]: {
```

Comment:
Big fan of the way this navigation config has been setup!  Makes it so easy to see which route has which referers and which roles are allowed.